### DISCUSSION TITLE
FastAPI Docs & Translations Management

### DISCUSSION THREAD

**Discussion by ceb10n** [fastapi/fastapi#11930] [Show and tell]
Hi 😄 

I created a simple cli based on typer to manage fastapi docs and translations.

It is inspired by the this PR #11864 from [@lucasbalieiro](https://github.com/lucasbalieiro) ❤️ 

Although it is in early development, it should be very straightforward to use: just install it and run:

```shell
pip install -U fastapi-translations
cd you-fastapi-local-repo
fastapi_translations -l pt -c
```

And it will create a summary and dump informations about the docs and translations of the given language into a csv:

![example](https://github.com/user-attachments/assets/09b528eb-1650-4e15-a7de-5f67885a5b6f)

If the PR gets accepted someday, we can move the features of this cli to fastapi repo

Links:

* https://github.com/ceb10n/fastapi-translations-management
* https://pypi.org/project/fastapi-translations/

**Comment by lucasbalieiro** [comment]
Hi @ceb10n,

After looking at your project, I think we don't need to wait for my PR to be accepted. How about we close my PR and mention your package in the discussion instead?

Your project is more polished and has more cool functionalities than the original proposal. 

This way, we can move forward without waiting for my  PR  to be approved. We can just use your package. 

What do you think?

**Reply by ceb10n** [reply]
It's you call!

I think it's a nice feature to have in the project. 

But we can keep adding features in this cli, and if tiangolo think it's a good idea and accept the PR, we can bring the features to the project.
